app/controllers/api/messages_controller.rb

```diff
@@ -38,13 +38,18 @@ def message_drafts
         @message.add_cascading_tag(tag)
       end
 
+      unless @message.valid?(:validate_uuid_uniqueness)
+        @message.destroy
```

Comment:
toto je divne, ak to nie je validne tak by sa to ani nemalo vyrobit ci?

Reply:
Hej, presunula som vyssie pred save